modules/mongodb/mongodb.go

```diff
@@ -67,7 +87,16 @@ func (m *Mongo) Collect() map[string]int64 {
 		return nil
 	}
 
-	ms := m.serverStatusCollect()
+	ms := map[string]int64{}
+	for _, result := range []map[string]int64{
+		m.serverStatusCollect(),
+		m.dbStatsCollect(),
+	} {
+		for k, v := range result {
+			ms[k] = v
+		}
+	}
```

Comment:
I think we need to refactor it to

```go
if err := m.serverStatusCollect(ms); err != nil {
    ....
}
if err := m.dbStatsCollect(ms); err != nil {
    ....
}
```

We don't want to call "dbStatsCollect" if "serverStatusCollect" fails (mongodb is not responding).

Also, I think we need to name functions to "collectThis"/"collectThat".

Reply:
done